### PR TITLE
rearrange products in nav

### PR DESCRIPTION
**Pull request by CalebZoe**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by CalebZoe** [comment]
@hargunkaur286

**Comment by l5io** [comment]
🚀 Preview for commit 8a532ccb15d85be48ec1c964fb6558320a8ed296 at: https://672dfc0eb04b025b0e9d5912--layer5.netlify.app

**Review by hargunkaur286 — APPROVED** [review]

**Comment by CalebZoe** [comment]
fixing broken links..

File: src/sections/General/Navigation/utility/menu-items.js [hunk 1]
```diff
@@ -84,41 +84,46 @@ const Data = {
...
-          path: "/cloud-native-management/meshery",
+          name: "Kanvas",
+          path: "/cloud-native-management/kanvas",
           sepLine: true
         },
         {
-          name: "Kanvas",
-          path: "/cloud-native-management/kanvas",
+          name: "Designer",
+          path: "/cloud-native-management/design",
```
Comment (CalebZoe) [inline src/sections/General/Navigation/utility/menu-items.js:93]:
```suggestion
          path: "/cloud-native-management/kanvas/design",
```

File: src/sections/General/Navigation/utility/menu-items.js [hunk 2]
```diff
@@ -84,41 +84,46 @@ const Data = {
...
+        },
+        {
+          name: "Operator",
+          path: "/cloud-native-management/operate",
```
Comment (CalebZoe) [inline src/sections/General/Navigation/utility/menu-items.js:97]:
```suggestion
          path: "/cloud-native-management/kanvas/operate",
```

**Comment by l5io** [comment]
🚀 Preview for commit 99c78e688ac0cc86d08d1e6192e4ccc747f0311f at: https://672e0d912c0c368ffda31516--layer5.netlify.app

**Pull request merged**